test/utils.sh

```diff
@@ -973,3 +973,27 @@ get_highest_bundle_version() {
 
   echo "$bundle_image"
 }
+
+# This function will be used by tasks in tekton-integration-catalog
+# Given the output of 'opm render $bundle_image' command, this function returns the supported architectures for the bundle
+get_bundle_arches() {
+  local RENDER_OUT_BUNDLE="$1"
+  local arches
+
+  # Validate input parameters
+  if [[ -z "$RENDER_OUT_BUNDLE" ]]; then
+    echo "get_bundle_arches: Invalid input. Usage: get_bundle_arches <RENDER_OUT_BUNDLE>" >&2
+    exit 2
+  fi
+
+  arches=$(echo "$RENDER_OUT_BUNDLE" | jq -cs \
```

Comment:
you might want to retain ` tr -d '\000-\031'` to err on the side of caution. I encountered a situation with some FBC fragments where the render out had control characters.

Reply:
thanks! added

Reply:
Should we add this as a common post-process for the `render_opm` command?